code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -120,7 +120,9 @@
 /obj/item/weapon/razor/attack(mob/M, mob/user)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
-
+		if(H.dna.species.id != "human")
+			user << "<span class='warning'>There is nothing to shave!</span>"
+			return

```

Comment:
Use

if(!(HAIR in H.dna.species.specflags) || !(FACEHAIR in H.dna.species.specflags)) //Might be a better way to phrase that

There is a mutant race that has hair (slimepeople) and it's just more futureproof in general